tox.ini

```diff
@@ -0,0 +1,68 @@
+[tox]
+skipsdist = True
+usedevelop = True
+
+envlist =
+    py{38,39,310}-django{32,41}-wagtail{41,42,50,51,52}-{sqlite,postgresql}
+    py311-django41-wagtail{41,42,50,51,52}-{sqlite,postgresql}
+    py311-django42-wagtail{50,51,52}-{sqlite,postgresql}
+    py312-django42-wagtail{52}-{sqlite,postgresql}
```

Comment:
Shall we drop the 3.12 testing for the moment?

Reply:
Dropped this along with Wagtail 4.2 and 5.0 due to EOL and since this is a Torchbox-maintained repo.